### DISCUSSION TITLE
How to open QSelect always drop down

### DISCUSSION THREAD

**Discussion by hoangphuckt7** [quasarframework/quasar#13508] [Ideas / Proposals]
![image](https://user-images.githubusercontent.com/81405523/170012955-db4f508e-5a31-407b-99f2-d0d47512e360.png)
I want q-select to always show the item below the input
![image](https://user-images.githubusercontent.com/81405523/170014022-84022038-df45-4ad2-a123-da9612484b55.png)
but when I pulled the mouse they showed up above the input
Please help me !

**Comment by MilosPaunovic** [comment]
Take note of `menu-anchor` prop.

[https://codepen.io/milospaunovic/pen/MWQEEdE?editors=1010](https://codepen.io/milospaunovic/pen/MWQEEdE?editors=1010)

**Reply by hoangphuckt7** [reply]
sorry, but i mean i want the menu list to be always at the bottom

**Comment by suleimanas** [comment]
As @MilosPaunovic said check out menu-anchor and menu-self props, but know it's bad behavior.
I know it's tempting to fix it but it's better to be responsive

**Comment by adrianlungu** [comment]
As much as responsiveness is appreciated, there are cases when it would be helpful to have it not be responsive in the way it is now (or allow turning it on and off), i.e.

https://codepen.io/adyjayex/pen/VwoydQG

If, for example, you have a top menu bar, and you open a dialog with a select inside with a lot of items, it will open it upward and in some cases, the values will be hidden under the menu bar, while having it downward would be no issue. 

Anchoring in this case changes nothing.
